apps/api/OsmoX-API.postman_collection.json

```diff
@@ -1000,6 +1000,599 @@
 				}
 			],
 			"description": "These API calls are responsible for all operations related to Server API Keys."
+		},
+		{
+			"name": "Provider Direct Requests",
+			"item": [
+				{
+					"name": "Mailgun cURL",
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "password",
+									"value": "{{MAILGUN_API_KEY}}",
+									"type": "string"
+								},
+								{
+									"key": "username",
+									"value": "api",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "formdata",
+							"formdata": [
+								{
+									"key": "from",
+									"value": "Excited User <postmaster@{{MAILGUN_DOMAIN}}>",
+									"type": "text"
```

Comment:
Since we are adding variables, you need to update Environment files too, and add default value for environment which won't change much.